src/components/common/icons/KeyholeIcon/index.tsx

```diff
@@ -0,0 +1,10 @@
+import Keyhole from '@/components/common/icons/KeyholeIcon/keyhole.svg'
+import type { ComponentProps } from 'react'
+
+import CircularIcon from '../CircularIcon'
+
+const KeyholeIcon = (props: Pick<ComponentProps<typeof CircularIcon>, 'height' | 'width' | 'variant'>) => {
+  return <CircularIcon component={Keyhole} alt="Not connected" color="error" {...props} />
+}
```

Comment:
This seems a bit much for something we could have 2 svgs for:

Keyhole:
```
<svg width="60" height="60" viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg">
<circle cx="30" cy="30" r="30" fill="#E8E7E6"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M27.031 43.575H34.3407C35.0771 43.575 35.6741 42.9781 35.6741 42.2417C35.6741 42.17 35.6683 42.0985 35.6568 42.0278L33.7781 30.4703C36.1541 29.3307 37.8041 26.9374 37.8041 24.1567C37.8041 20.2858 34.619 17.1431 30.6878 17.1431C26.7596 17.1431 23.5715 20.2858 23.5715 24.1567C23.5672 26.9402 25.2144 29.3307 27.5961 30.4703L25.715 42.0275C25.5967 42.7543 26.09 43.4394 26.8168 43.5577C26.8876 43.5692 26.9593 43.575 27.031 43.575Z" fill="#B2B5B2"/>
<circle cx="52" cy="52" r="7" fill="#F02525" stroke="white" stroke-width="2"/>
</svg>
```

Warning:
```
<svg xmlns="http://www.w3.org/2000/svg" width="90" height="90" viewBox="0 0 90 90"><g fill="none" fill-rule="evenodd"><circle cx="45" cy="45" r="45" fill="#E8E7E6"></circle><path fill="#B2B5B2" fill-rule="nonzero" d="M26.24 62c-.57 0-1.13-.15-1.623-.437a3.254 3.254 0 0 1-1.18-4.44l18.76-32.501a3.234 3.234 0 0 1 5.606 0l18.761 32.501c.286.495.436 1.057.436 1.628A3.244 3.244 0 0 1 63.76 62H26.24z"></path><path fill="#E8E7E6" fill-rule="nonzero" d="M47 49h-4V36h4zM47 55h-4v-4h4z"></path><circle cx="78" cy="78" r="11" fill="#FFC05F" fill-rule="nonzero" stroke="#F6F7F8" stroke-width="4"></circle></g></svg>
```